src/pages/Profile.js

```diff
@@ -174,7 +174,11 @@ const FenceLink = ({ provider, displayName }) => {
   /*
    * Hooks
    */
-  const [{ username, issued_at: issuedAt }, setStatus] = useState({})
+  const auth = Utils.useStore(authStore)
+  const { username, issued_at: issuedAt } = Utils.switchCase(provider,
+    ['fence', () => auth.fenceDCPStatus || {}],
+    ['dcf-fence', () => auth.fenceDCFStatus || {}],
+    [Utils.DEFAULT, () => {}])
```

Comment:
This `Utils.switchCase` will need to be updated when merging with #2161. I suspect the result will also be simplified... possibly removing the `switchCase` altogether.

Reply:
TOL: It is interesting that we have an explicit DEFAULT option for switchCase, but not for cond (it just takes a function as its last arg). I wonder if we should bring them into alignment one way or the other? (I kinda like having the default there explicitly?)

Reply:
I've wondered about that, too.

It could just be how they came to be. I think `cond` is inspired by Clojure (?) where the default is an optional last arg while `switchCase` is inspired by the `switch` statement which explicitly declares the default.

Even so, I think I'm also in favor of standardizing, and I don't have a strong preference either way.

Reply:
FWIW I've never loved the bare function as a default case for `cond`. A more consistent form would be an always-true clause like `[true, () => blah]`, which would work without any special handling. We could make that change as long as we visited every caller and updated them.